configs/variables

```diff
@@ -1,5 +1,3 @@
-#!/bin/bash
-
```

Comment:
I deleted it because the variables file is sourced in other scripts, not executed on it's own. I don't think having the shebang there really has any effect outside of making shellcheck happier, but I was banging my head at a variable scope thing which turned out to just be me misremembering how bash exports work. I'll add it back.